hadoop-ozone/dev-support/intellij/ozone-site.xml

```diff
@@ -71,4 +71,4 @@
     <name>datanode.replication.port</name>
     <value>0</value>
   </property>
-</configuration>
+</configuration>
```

Comment:
nit: Can we remove this change since it has no effect and is not related to the task?